agent/consul/rate/handler_test.go

```diff
@@ -0,0 +1,105 @@
+package rate
+
+import (
+	"testing"
+
+	"github.com/hashicorp/consul/agent/consul/multilimiter"
+	mock "github.com/stretchr/testify/mock"
+)
+
+func TestNewHandlerWithLimiter_CallsUpdateConfig(t *testing.T) {
+	mockRateLimiter := multilimiter.NewMockRateLimiter(t)
+	mockRateLimiter.On("UpdateConfig", mock.Anything, mock.Anything).Return(
+		func(c multilimiter.LimiterConfig, prefix []byte) {})
```

Comment:
I don't think `UpdateConfig` returns anything. Would this do the trick?

```suggestion
	mockRateLimiter.On("UpdateConfig", mock.Anything, mock.Anything).Return()
```

Reply:
Thank you. I updated this as well as another mock instance in the file.